src/main/scala/ing/wbaa/druid/client/RequestFlowExtension.scala

```diff
@@ -0,0 +1,76 @@
+package ing.wbaa.druid.client
```

Comment:
Here as well, please add the header: Please add the Apache 2.0 license header: https://github.com/ing-bank/scruid/blob/master/src/main/scala/ing/wbaa/druid/DruidConfig.scala#L1-L16